cloudbuild.yaml

```diff
@@ -0,0 +1,21 @@
+steps:
+        - name: gcr.io/cloud-builders/docker
+          args:
+                - 'build'
+                - '--tag=%REPO%/mkdebootstrap:%VERSION%'
+                - 'mkdebootstrap/'
+        - name: gcr.io/cloud-builders/docker
+          args:
+                - 'build'
+                - '--tag=%REPO%/docker-mkdebootstrap:%VERSION%'
```

Comment:
This image is an intermediate one, right? As in we don't need to push it? If so, could you drop the %REPO% and %VERSION% variables to make it more clear? A comment might help too.